erpnext/hr/doctype/employee/employee.py

```diff
@@ -257,46 +257,24 @@ def send_birthday_reminders():
 		 	}
 		)
 		employee_emails = get_employee_emails(employee_list)
-		birthday_names = [name["employee_name"] for name in birthdays]
-		birthday_emails = [email["user_id"] or email["personal_email"] or email["company_email"] for email in birthdays]
-
-		birthdays.append({'company_email': '','employee_name': '','personal_email': '','user_id': ''})
-
-		for e in birthdays:
-			if e['company_email'] or e['personal_email'] or e['user_id']:
-				if len(birthday_names) == 1:
-					continue
-				recipients = e['company_email'] or e['personal_email'] or e['user_id']
-
-
+		
+		for birthday in birthdays:
+			if frappe.db.exists("Email Template", frappe.db.get_single_value("HR Settings", "birthday_email_template")):
+				email_template = frappe.get_doc("Email Template", frappe.db.get_single_value("HR Settings", "birthday_email_template")).response
 			else:
-				recipients = list(set(employee_emails) - set(birthday_emails))
+				email_template = "Happy Birthday {0}! \U0001F603".format(birthday["employee_name"])
```

Comment:
This is not the `email_template`, this is the direct `message`. I think this will break when you do the `frappe.render_template` with this string.

Also, use `dict.get(key)` instead of `dict[key]` to avoid KeyError.

Reply:
The message string doesn't break with `frappe.render_template`.

Reply:
It's still an unnecessary call though. Just do something like:

```python
if template_exists:
	template = get_template()
	message = render_template(template, context)
else:
	message = "default_message"
```

And pass the `message` to `sendmail`.